package.json

```diff
@@ -59,13 +67,20 @@
   "ava": {
     "verbose": true,
     "files": [
-      "test/**/*.test.js"
+      "test/**/*.test.ts"
+    ],
+    "require": [
+      "ts-node/register",
+      "source-map-support/register"
+    ],
+    "extensions": [
+      "ts"
     ]
   },
   "publishConfig": {
     "access": "public"
   },
   "peerDependencies": {
-    "aws-xray-sdk-core": "^2.0.0"
+    "aws-xray-sdk-core": "^2.0.0 || ^3.0.0"
```

Comment:
oh, interesting, I didn't realize it was possible to specify dependencies like this